src/main/groovy/io/seqera/wave/service/blob/impl/BlobCacheServiceImpl.groovy

```diff
@@ -193,55 +178,20 @@ class BlobCacheServiceImpl implements BlobCacheService {
         return command
     }
 
-    protected BlobCacheInfo store(RoutePath route, BlobCacheInfo info) {
-        BlobCacheInfo result
-        try {
-            log.debug "== Blob cache begin for object '${info.locationUri}'"
-            result = store0(route, info)
-            //check if the cached blob size is correct
-            result = checkUploadedBlobSize(result, route)
-        }
-        finally {
-            // use a short time-to-live for failed downloads
-            // this is needed to allow re-try caching of failure transfers
-            final ttl = result.succeeded()
-                    ? blobConfig.statusDuration
-                    : blobConfig.failureDuration
-
-            blobStore.storeBlob(route.targetPath, result, ttl)
-            return result
-        }
-    }
-
-    /**
-     * Check the size of the blob stored in the cache
-     *
-     * @return {@link BlobCacheInfo} the blob cache info
-     */
-    protected BlobCacheInfo checkUploadedBlobSize(BlobCacheInfo info, RoutePath route) {
```

Comment:
no blob size checking anymore?

Reply:
This is useless. Both S3 and R2 implements strong consistency (upload cannot be partial)

https://developers.cloudflare.com/r2/reference/consistency/#operations-and-consistency

Reply:
ok understood